packages/next-app/src/components/Table.tsx

```diff
@@ -98,9 +98,20 @@ function Body({ children }: React.PropsWithChildren) {
 function BodyRow({
   children,
   classNames,
-}: React.PropsWithChildren<{ classNames?: string }>) {
+  onClick,
+}: React.PropsWithChildren<{ classNames?: string; onClick?: () => void }>) {
   useTableContext();
-  return <tr className={cn(classNames ?? classNames)}>{children}</tr>;
+  return (
+    <>
+      {onClick ? (
+        <tr className={cn(classNames ?? classNames)} onClick={onClick}>
```

Comment:
`classNames ?? classNames` shouldn't be needed since `cn` [can take an undefined string](https://github.com/JedWatson/classnames/blob/main/index.d.ts).